diavlos/src/bpmn/bpmn.py

```diff
@@ -8,13 +7,15 @@
 _TITLE_STR = 'title'
 _IMPLEMENTATION_STR = 'implementation'
 _CHILD_STR = 'child'
+_EXIT_STR = 'exit'
 _SEMANTIC_STR = 'semantic'
 _DEFINITIONS_STR = 'definitions'
 _TIMER_STR = 'Timer'
 _MIN_STR = 'min'
 _MAX_STR = 'max'
 _NUM_ID_SUFFIX = 'num_id'
 _PREV_CHILD_SUFFIX = f'previous_{_CHILD_STR}'
+_lifeline = 0  # is used for the correct alignment of substeps and their substeps
```

Comment:
This variable is never used. If it is, it should be uppercase (PEP8): `_LIFELINE`